nodewatcher/modules/devices/ubnt/unifi.py

```diff
@@ -0,0 +1,145 @@
+from django.utils.translation import ugettext_lazy as _
+
+from nodewatcher.core.generator.cgm import base as cgm_base, protocols as cgm_protocols, devices as cgm_devices
+
+
+class UBNTUapACLite(cgm_devices.DeviceBase):
+    """
+    UBNT UAP AC Lite device descriptor.
+    """
+
+    identifier = 'ub-uap-ac-lite'
+    name = "Unifi AP AC Lite"
+    manufacturer = "Ubiquiti"
+    url = 'http://www.ubnt.com/'
+    architecture = 'ar71xx'
+    radios = [
+        cgm_devices.IntegratedRadio('wifi0', _("Integrated wireless radio (5 GHz)"), [
+            cgm_protocols.IEEE80211AC(
+                cgm_protocols.IEEE80211AC.SHORT_GI_20,
+                cgm_protocols.IEEE80211AC.SHORT_GI_40,
+                cgm_protocols.IEEE80211AC.RX_STBC1,
+                cgm_protocols.IEEE80211AC.DSSS_CCK_40,
+            )
+        ], [
+            cgm_devices.AntennaConnector('a1', "Antenna1")
```

Comment:
I have already stated that.
Difference is in the construction of the antennas,Lite has 2x2 MIMO for 2.4Ghz and another 2x2 MIMO for 5GHz while Pro has 3x3 MIMO for 2.4GHz and another one 3x3 MIMO for 5Ghz